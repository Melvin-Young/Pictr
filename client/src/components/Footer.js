import React from 'react';

const Footer = () => (
	<footer className="footer">
		<div className="footer-container">
			<span>By Melvin Young :) </span>
		</div>
	</footer>
);

export default Footer;
