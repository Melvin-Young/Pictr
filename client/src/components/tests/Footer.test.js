import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import Footer from '../Footer';

Enzyme.configure({ adapter: new Adapter()});

describe('Footer', () => {
    it('should render', () => {
        const component = shallow(<Footer />)
        expect(component).toMatchSnapshot();
    });
});
