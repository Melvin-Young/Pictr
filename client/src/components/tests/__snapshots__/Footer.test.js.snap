// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="footer-container"
      >
        <span>
          By Melvin Young :) 
        </span>
      </div>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <span>
          By Melvin Young :) 
        </span>,
        "className": "footer-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "By Melvin Young :) ",
        },
        "ref": null,
        "rendered": "By Melvin Young :) ",
        "type": "span",
      },
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="footer-container"
        >
          <span>
            By Melvin Young :) 
          </span>
        </div>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <span>
            By Melvin Young :) 
          </span>,
          "className": "footer-container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "By Melvin Young :) ",
          },
          "ref": null,
          "rendered": "By Melvin Young :) ",
          "type": "span",
        },
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
